frontend/src/components/Input/index.js

```diff
@@ -0,0 +1,43 @@
+import { PropTypes } from "prop-types";
+import styles from "./index.module.css";
+
+const Input = ({ inputType, disabled, error }) => {
+  let texts = {
+    text: { placeholder: "Ariana", errorMessage: "Enter your name." },
+    email: {
+      placeholder: "arina@fortunemail.com",
+      errorMessage: "Invalid email.",
+    },
+  };
```

Comment:
If it's possible, I would prefer to see these used directly. I find this a bit confusing.

Reply:
Perhaps you can look into [default prop values](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) ...

I think `disabled` should default to `false`

Reply:
Agree, this shouldn't be here it should be configured by props

Reply:
I removed the texts object and I added this:
```
Input.defaultProps = {
  disabled: false,
};
```